src/lab15.py

```diff
@@ -17,29 +17,35 @@
 import time
 import urllib.parse
 import wbetools
+import OpenGL.GL
 
 from lab1 import parse_url
+from lab2 import WIDTH, HEIGHT, HSTEP, VSTEP, SCROLL_STEP
 from lab4 import print_tree
-from lab13 import Text, Element
+from lab5 import BLOCK_ELEMENTS
+from lab14 import Text, Element
```

Comment:
This is out of order, is it necessary for patching reasons?

Reply:
No, just matching the existing order.